pkg/eth/rpc_types.go

```diff
@@ -199,55 +200,87 @@ func (r *GetLogsRequest) UnmarshalJSON(data []byte) error {
 
 // ========== GetTransactionByHash ============= //
 type (
+	// Presents transacrion hash value
```

Comment:
transaction* spelling

Reply:
Issue is resolved, see commit by hash: ac5dbdf7e91c4141ad11b2fd7d50b95f22609a9d